charts/payment-integration/templates/service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Chart.Name }}
+  labels:
+    app: {{ .Chart.Name }}
```

Comment:
use same label across all k8s resources in this chart:
```
labels:
  app: {{ template "payment.name" . }}
  chart: {{ template "payment.chart" . }}
  release: {{ .Release.Name }}
  heritage: {{ .Release.Service }}
```

Reply:
see https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/labels.md#standard-labels